js/index.js

```diff
@@ -39,4 +39,75 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+const navA = document.querySelectorAll('a');
```

Comment:
Good job coming up with a solution for this but where is the code for turning the nav green?